ksql-parser/src/main/java/io/confluent/ksql/parser/json/ColumnRefSerializationModule.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.parser.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import io.confluent.ksql.execution.expression.tree.ColumnReferenceExp;
+import io.confluent.ksql.execution.expression.tree.Expression;
+import io.confluent.ksql.parser.ExpressionParser;
+import io.confluent.ksql.schema.ksql.ColumnRef;
+import io.confluent.ksql.schema.ksql.FormatOptions;
+import io.confluent.ksql.util.IdentifierUtil;
+import java.io.IOException;
+
+public class ColumnRefSerializationModule extends SimpleModule {
```

Comment:
Why do we need all these different modules?  Why not just have a single module that adds all the types you need? e.g. `KsqlParserSerializationModule`.

then I'd put each serializer and deserializer in its own class, or maybe one class per pair with nested classes inside.  

Or, another thought... we could just annotate our types with the Jackon annotations.  This has the benefit that it makes it clear to anyone working on the types that they are being serialized and any changes need to be compatible.

Reply:
I don't want to annotate because the types shouldn't depend on the parser. I don't really have  a strong preference for one module vs multiple. I liked having them separate because there's not really any need to group them, and keeping them separate keeps things flexible. I don't think it's a big deal though, so happy to change it to a common module.

Reply:
I don't feel strongly about the multiple vs single modules

Reply:
Eh I've already changed it now :)